docs/end-user-guide/using-jira-commands.md

```diff
@@ -1,60 +1,34 @@
-* `/jira instance list` - List installed Jira instances
 * `/jira connect [jiraURL]` - Connect your Mattermost account to your Jira account
 * `/jira disconnect [jiraURL]` - Disconnect your Mattermost account from your Jira account
-* `/jira [issue] assign [issue-key] [assignee]` - Change the assignee of a Jira issue
-* `/jira [issue] create [text]` - Create a new Issue with 'text' inserted into the description field
-* `/jira [issue] transition [issue-key] [state]` - Change the state of a Jira issue
-* `/jira [issue] unassign [issue-key]` - Unassign the Jira issue
-* `/jira [issue] view [issue-key]` - View the details of a specific Jira issue
+* `/jira issue assign [issue-key] [assignee]` - Change the assignee of a Jira issue
+* `/jira issue create [text]` - Create a new Issue with 'text' inserted into the description field
+* `/jira issue transition [issue-key] [state]` - Change the state of a Jira issue
+* `/jira issue unassign [issue-key]` - Unassign the Jira issue
+* `/jira issue view [issue-key]` - View the details of a specific Jira issue
+* `/jira instance settings` - View your user settings
 * `/jira instance settings [setting] [value]` - Update your user settings
 
-Then select the project and issue type, add a summary, and a description.
-
-![image](https://user-images.githubusercontent.com/13119842/59113188-985a9280-8912-11e9-9def-9a7382b4137e.png)
-
-Click **Create** to create the issue which includes any file attachments that were part of the Mattermost message.
-
-![image](https://user-images.githubusercontent.com/13119842/59113219-a4deeb00-8912-11e9-9741-5ddc8a4b51fa.png)
-
-**Note:** This plugin does not support all Jira fields. If the project you tried to create an issue for has **required fields** not yet supported, you'll be prompted to manually create an issue. Clicking the provided link opens the issue creation screen on the Jira web interface. The information you entered in Mattermost is migrated over so no work is lost.
-
-The supported Jira fields are:
-
-* **Project Picker:** Custom fields and the built-in **Project** field.
-* **Single-Line Text:** Custom fields, and built-in fields such as **Summary** and **Environment**.
-* **Multi-Line Text:** Custom fields, and built-in fields such as **Description**.
-* **Single-Choice Issue:** Custom fields, and built-in fields such as **Issue Type** and **Priority**. 
-* **Assignee:** System field.
-
-### Attach Messages to Jira issues
+**Note:** For the `/jira instance settings` command, [setting] can be `notifications` and [value] can be `on` or `off`
 
-Keep all information in one place by attaching parts of Mattermost conversations in Jira issues as comments. To attach a message, hover over the relevant message and select **\(...\) > More Actions > Attach to Jira Issue**.
+### Authenticating with Jira
 
-![You can attach a message to an existing Jira ticket](../.gitbook/assets/image%20%286%29.png)
```

Comment:
@mickmister, removing this image is what I was referring to.  Is it referenced anywhere else or can the file be deleted?

Reply:
@jfrerich That linked image was moved to another file, `docs/end-user-guide/interacting-with-forms.md`. Notice how the original page is called `using-jira-commands`, which has nothing to do with UI features of the plugin, including the "Attach to Issue" feature, hence this PR being created 🙂 

Take a look at https://mattermost.gitbook.io/plugin-jira/end-user-guide/using-jira-commands and notice how the first section is about commands but then has UI-related info shuffled within the text. Also the actual image doesn't match the Attach feature, hence this PR https://github.com/mattermost/mattermost-plugin-jira/pull/782